main.go

```diff
@@ -54,6 +55,7 @@ func init() {
 	utilruntime.Must(policyv1beta1.AddToScheme(scheme))
 	utilruntime.Must(rbacv1.AddToScheme(scheme))
 
+	utilruntime.Must(metallbiov1alpha1.AddToScheme(scheme))
```

Comment:
same, this is already added to the scheme I think?

Reply:
same outdated